pkgs/by-name/js/jsonschema-cli/package.nix

```diff
@@ -0,0 +1,66 @@
+{
+  lib,
+  fetchCrate,
+  rustPlatform,
+  versionCheckHook,
+  nix-update-script,
+  runCommand,
+  testers,
+  jsonschema-cli,
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "jsonschema-cli";
+  version = "0.29.0";
+
+  src = fetchCrate {
+    inherit pname version;
+    hash = "sha256-kpdvvCnMsHfogXmAqNeo1Cl1hZtCPHqkfhYm8ipWToo=";
+  };
```

Comment:
Consider fetching the source from GitHub. Afaik using Github is prefered

Reply:
I have preferred using crate because of currently the GitHub repository does not save the Cargo.lock. Are there any documents explaining how to determine these methods?

Reply:
Ok, this is a good reason to use the crate